package/src/views/useTouchHandler.ts

```diff
@@ -41,14 +42,16 @@ export const useTouchHandler = (handlers: TouchHandlers): TouchHandler => {
           touch.type !== TouchType.End &&
           touch.type !== TouchType.Cancelled
         ) {
-          prevvelocityRef.current.x = distX / Math.max(0.0001, timeDiffseconds);
-          prevvelocityRef.current.y = distY / Math.max(0.0001, timeDiffseconds);
+          prevvelocityRef.current.x =
```

Comment:
Is the 0 case handled properly? here a timeDiffSeconds of would generate a huge velocity but it probably means that we should return the already computed velocity?